### ISSUE TITLE
Tags `sidewalk:right` and `sidewalk:left` not supported

### ISSUE THREAD

**Issue by M-Miller-42** [abrensch/brouter#667]
As far as I can see, sidewalks tagged with `sidewalk:left=yes` or are not considered for routing (opposed to `sidewalk=left`, which works fine). 
Here is an [example](https://brouter.de/brouter-web/#map=19/48.13290/11.57297/standard&lonlats=11.573303,48.133296;11.572469,48.132823&profile=hiking-mountain) where this leads to a detour, because the secondary highway with _seemingly_ no sidewalk is avoided.
The [wiki](https://wiki.openstreetmap.org/wiki/Key:sidewalk:right) lists the three possible values `no`, `yes`, `separate` and has some hints for data consumers, that might be useful. 

Of course the same situation applies for  `sidewalk:right` tags.

Thank you for this great project, I am using it daily.

**Comment by afischerdev** [comment]
Yes, `sidewalk:left=yes` is not a part of the lookups.dat [current version](https://github.com/abrensch/brouter/blob/master/misc/profiles2/lookups.dat) and the future version. #458  [lookups.dat future](https://github.com/afischerdev/brouter/blob/update-lookups/misc/profiles2/lookups.dat).
I'll add that.

**Comment by afischerdev** [comment]
Added sidewalk to #458 lookups.dat

**Comment by torotil** [comment]
I have observed a similar issue with `sidewalk:right=yes` on a one-way primary highway. In my [example](https://brouter.de/brouter-web/#map=19/48.22714/16.36440/standard,route-quality&lonlats=16.365007,48.227149;16.364093,48.227304) it routes around the block instead of pushing the bike on the sidewalk against the one-way.
